mappings/net/minecraft/server/function/FunctionLoader.mapping

```diff
@@ -7,13 +7,18 @@ CLASS net/minecraft/class_5349 net/minecraft/server/function/FunctionLoader
 	FIELD field_25331 level I
 	FIELD field_25332 commandDispatcher Lcom/mojang/brigadier/CommandDispatcher;
 	FIELD field_25801 tags Lnet/minecraft/class_5414;
+	FIELD field_33385 PATH_SUFFIX Ljava/lang/String;
```

Comment:
Just `EXTENSION` since this is not likely to conflict with other extension types. In addition, need to rename `PATH_SUFFIX_LENGTH` to `EXTENSION_LENGTH`